src/components/WwwFrame/Header/KvAtbModalContainer.vue

```diff
@@ -241,23 +242,44 @@ const pillMsg = computed(() => {
 		return 'You’re close to your next milestone!';
 	}
 
-	const milestonesCopy = contributingAchievements.value.length > 1
-		? `${contributingAchievements.value.length} of your milestones`
+	const milestonesCopy = achievementsFromBasket.value.length > 1
+		? `${achievementsFromBasket.value.length} of your milestones`
 		: 'your next milestone';
 
 	return borrowerName.value
 		? `Supporting ${borrowerName.value} will hit ${milestonesCopy}!`
 		: 'Supporting this loan achieves a milestone!';
 });
 
+const updateTierTable = () => {
+	achievementsFromBasket.value.forEach(achievement => {
+		tierTable.value[achievement.achievementId] = achievement.postCheckoutTier;
+	});
+};
+
 const fetchPostCheckoutAchievements = async loanIds => {
 	await apollo.query({
 		query: postCheckoutAchievementsQuery,
 		variables: { loanIds }
 	}).then(({ data }) => {
 		const loanAchievements = data.postCheckoutAchievements?.overallProgress ?? [];
-		contributingAchievements.value = loanAchievements.filter(achievement => achievement.postCheckoutTier !== achievement.preCheckoutTier); // eslint-disable-line max-len
-		const nonContributingAchievements = loanAchievements.filter(achievement => achievement.postCheckoutTier === achievement.preCheckoutTier); // eslint-disable-line max-len
+		const contributingLoanAchievements = [];
```

Comment:
The new changes are looking good, but could you add a new ticket to the sprint and epic to extract out this logic and write unit testing? Ideally we pull out ~line266-334 and write testing for all expected user paths/cases. There's enough going on here that I think we want to stabilize it with testing. Once you create the ticket, please add a comment to this thread so we can see the follow-up from this PR. Thanks!